### ISSUE TITLE
i want train hg_2，about loss

### ISSUE THREAD

**Issue by following-up** [Elody-07/AWR-Adaptive-Weighting-Regression#19]
in train.py 121:
       loss = (loss_coord + loss_offset)

if i want to try hg_2, loss should be set 
       loss+ = (loss_coord + loss_offset)

should i do this？PLZ
